services/minespace-web/src/components/dashboard/mine/overview/Overview.js

```diff
@@ -42,145 +44,206 @@ export class Overview extends Component {
+              {(this.props.mine.mine_location && this.props.mine.mine_location.latitude) ||
+                Strings.UNKNOWN}
+            </Descriptions.Item>
+            <Descriptions.Item label="Longitude">
+              {(this.props.mine.mine_location && this.props.mine.mine_location.longitude) ||
+                Strings.UNKNOWN}
+            </Descriptions.Item>
+            <Descriptions.Item span={2} label="Operating Status">
+              {(this.props.mine.mine_status &&
+                this.props.mine.mine_status.length > 0 &&
+                this.props.mine.mine_status[0].status_labels.join(", ")) ||
+                Strings.UNKNOWN}
+            </Descriptions.Item>
+            <Descriptions.Item span={2} label="Commodity">
+              {this.props.transformedMineTypes.mine_commodity_code &&
+              this.props.transformedMineTypes.mine_commodity_code.length > 0
+                ? this.props.transformedMineTypes.mine_commodity_code
+                    .map((code) => this.props.mineCommodityOptionsHash[code])
+                    .join(", ")
+                : Strings.UNKNOWN}
+            </Descriptions.Item>
+            <Descriptions.Item span={2} label="Disturbance">
+              {this.props.transformedMineTypes.mine_disturbance_code &&
+              this.props.transformedMineTypes.mine_disturbance_code.length > 0
+                ? this.props.transformedMineTypes.mine_disturbance_code
+                    .map((code) => this.props.mineDisturbanceOptionsHash[code])
+                    .join(", ")
+                : Strings.UNKNOWN}
+            </Descriptions.Item>
+            <Descriptions.Item span={2} label="Active Permits">
+              {this.props.mine.mine_permit_numbers && this.props.mine.mine_permit_numbers.length > 0
+                ? this.props.mine.mine_permit_numbers.join(", ")
+                : Strings.NONE}
             </Descriptions.Item>
-            <Descriptions.Item label="Commodity">{Strings.NOT_APPLICABLE}</Descriptions.Item>
-            <Descriptions.Item label="Operating Status">{Strings.NOT_APPLICABLE}</Descriptions.Item>
-            <Descriptions.Item label="Disturbance">{Strings.NOT_APPLICABLE}</Descriptions.Item>
           </Descriptions>
-          <Row>
+          <Row gutter={[16, 16]}>
             <Col xl={{ span: 11 }} xxl={{ span: 10 }}>
               <ContactCard
                 title="Mine Manager"
-                party={partyMineManager}
+                party={mineManager}
                 dateLabel="Mine Manager Since"
               />
             </Col>
             {/* Disabled until we find a replacement contact to fill this card */}
             {/* <Col xl={{ span: 11, offset: 2 }} xxl={{ span: 10, offset: 2 }}>
-              <ContactCard title="Permittee" party={partyPermittee} dateLabel="Permittee Since" />
+              <ContactCard title="Permittee" party={null} dateLabel="Permittee Since" />
             </Col> */}
```

Comment:
This can be completely removed and readded when we know what will be going here, I don't think it will happen this sprint